src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -98,7 +98,7 @@ idna==2.8
 invoke==1.2.0
 isodate==0.6.0
 Jinja2==2.10.1
-jmespath==0.9.4
+jmespath==0.9.5
```

Comment:
I have also updated the requirements for other platforms. Also submitted PR https://github.com/microsoft/knack/pull/208.